pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -273,7 +267,7 @@ def _call_drain_handler(self):
 
         # set the flag to False so the drain handler will not be called more than once
         self._is_drain_needed = False
-        return self._platform._on_signal(callback_type="drain")
+        self._platform._on_signal(callback_type="drain")
```

Comment:
I think we can leave the `return` here due to https://github.com/nuclio/nuclio-sdk-py/pull/60/files .
It shouldn't really affect the caller flow

Reply:
@TomerShor then we will need awaits in finally, I'd be on safe side and include this after

Reply:
It really doesn't matter either way, because this method will now be called directly by the signal handler and anything it returns will be discarded.